recipe/meta.yaml

```
@@ -18,8 +18,14 @@ requirements:
     - {{ compiler('rust') }}
     - {{ compiler('c') }}
     - {{ stdlib("c") }}
+    - python >=3.10
     - nodejs
     - pnpm
+    - pkg-config
+  host:
+    - openssl
```

The macOS builds don't seem to link to it, so you should check if that is desired (IDK, didn't check) and if yes put this in as
```suggestion
    - openssl  # [not osx]
```
or
```suggestion
    - openssl  # [linux]
```